SDNExpress/scripts/SDNExpress.ps1

```diff
@@ -2401,6 +2401,215 @@ Configuration ConfigureHostAgent
     }
 }
 
+Configuration ConfigureIDns
+{
+    
+	Import-DscResource -ModuleName 'PSDesiredStateConfiguration'
+
+	Node $AllNodes.Where{$_.Role -eq "NetworkController"}.NodeName
+
+	{
+        
+		Script CreateiDnsCredentials
+
+		{
+       
+			SetScript = {
+
+	                . "$($using:node.InstallSrcDir)\Scripts\NetworkControllerRESTWrappers.ps1" -ComputerName $using:node.NetworkControllerRestName -UserName $using:node.NCClusterUserName -Password $using:node.NCClusterPassword
+
+                    	write-verbose "Adding iDNS server credentials. UserName=$($using:node.DomainJoinUsername) Password=$($using:node.DomainJoinPassword)"

```

Comment:
I'd need to check with MASD, but I think there was an effort to eliminate passwords being output into the script logs. It's ok to log that we are adding iDNS server credentials, but lets leave the actual creds out of the logging.